setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup, find_packages
+
+with open("README.md", 'r') as f:
+    long_description = f.read()
+
+setup(name='review-rot',
```

Comment:
I think you may need to rebase this branch. This file was included in https://github.com/nirzari/review-rot/pull/4